__tests__/unit/repositories/rule-flow.repository.test.ts

```
@@ -0,0 +1,670 @@
+
+      mockHandlePostExecuteSqlStatement.mockResolvedValue({
+        rows: [mockRuleFlow],
+        rowCount: 1,
+      });
+
+      const result = await findRuleFlowFromDB(mockRuleId, mockTenantId, '*');
+
+      expect(result).toEqual(mockRuleFlow);
+      expect(mockHandlePostExecuteSqlStatement).toHaveBeenCalledWith(
+        expect.objectContaining({
+          text: expect.stringContaining('SELECT *'),
```

Biome (2.4.14)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)